### ISSUE TITLE
Not smooth on sliding

### ISSUE THREAD

**Issue by owengombas** [archriss/react-native-image-gallery#99]
When I slide between images, the animation is not smooth (in production or debug mode)...  
I use Expo, I don't know if this is the problem

**Comment by luis-wang** [comment]
same problem

**Comment by 28harishkumar** [comment]
+1

**Comment by 28harishkumar** [comment]
When I `initialPage` and `onPageSelected` and binded initialPage with my parent state, it is slightly smooth (slightly not perfectly). This is still a bug.
react-native-cli: 2.0.1
react-native: 0.55.4

**Comment by elizond0** [comment]
any progress ?

**Comment by kukadiajayesh** [comment]
Yes, for the large size images it not perfectly smooth.

**Comment by srivatsav** [comment]
Yes @kukadiajayesh  is correct. Even I am experiencing the same issue for large size images.

**Comment by MursiDirect** [comment]
Any update yet, I still have the same issue?
